plugins/azuresso/azuresso.py

```diff
@@ -77,46 +77,46 @@ def azuresso_authenticate(url, username, password, useragent, pluginargs):
     headers = utils.add_custom_headers(pluginargs, headers)
 
     try:
-        r = requests.post("{url}/{domain}/winauth/trust/2005/usernamemixed?client-request-id={requestid}".format(url=url,domain=pluginargs['domain'],requestid=requestid), data=tempdata, headers=headers, verify=False, timeout=30)
+        r = requests.post(f"{url}/{pluginargs['domain']}/winauth/trust/2005/usernamemixed?client-request-id={requestid}", data=tempdata, headers=headers, verify=False, timeout=30)
 
         xmlresponse = str(r.content)
         creds = username + ":" + password
 
         # check our resopnse for error/response codes
         if "AADSTS50034" in xmlresponse:
-            data_response['output'] = "[-] FAILURE: Username not found - {}".format(creds)
+            data_response['output'] = f"[-] FAILURE: Username not found - {creds}"
             data_response['result'] = "failure"
 
         elif "AADSTS50126" in xmlresponse:
-            data_response['output'] = "[!] VALID_USERNAME - {} (invalid password)".format(creds)
+            data_response['output'] = f"[!] VALID_USERNAME - {creds} (invalid password)"
             data_response['result'] = "failure"
-            data_response['valid_user'] = True
 
         elif "DesktopSsoToken" in xmlresponse:
-            data_response['output'] = "[+] SUCCESS: {}".format(creds)
+            data_response['output'] = f"[+] SUCCESS: {creds}"
             data_response['result'] = "success"
             data_response['valid_user'] = True
 
             token = re.findall(r"<DesktopSsoToken>.{1,}</DesktopSsoToken>", xmlresponse)
             if (token):
-                data_response['output'] += " - GOT TOKEN {}".format(token[0])
+                data_response['output'] += f" - GOT TOKEN {token[0]}"
 
         elif "AADSTS50056" in xmlresponse:
-            data_response['output'] = "[!] VALID_USERNAME - {} (no password in AzureAD)".format(creds)
+            data_response['output'] = f"[!] VALID_USERNAME - {creds} (no password in AzureAD)"
             data_response['result'] = "failure"
-            data_response['valid_user'] = True
```

Comment:
add back in

Reply:
Add back in valid user tags for all plugins, used for automated logging of valid users